.changeset/unlucky-walls-tie.md

```diff
@@ -1,5 +1,7 @@
 ---
-"@coinbase/onchainkit": minor
+"@coinbase/onchainkit": patch
 ---
 
-**feat**: Swap success state - refetch balances and clear inputs by @0xAlec #1089
+- **patch**: refetch balances and clear inputs after Swap succeeds. By @0xAlec #1089
+- **patch**: Add connect wallet functionality to Swap component for disconnected user. By @abcrane123 #1173
+- **patch**: Adjust Swap component style to prevent UI shifting. By @abcrane123 #1184
```

Comment:
all sentences have to be in past tense, and start lowercase.

Reply:
changed in latest